documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -1196,11 +1197,11 @@ Key: User:user_001
 { 
   "addresses" : [
     { 
-      "addresses_KEY" : "work",
+      "addressType" : "work",
```

Comment:
Is that actually right? The example seems not to re-define the map key column? Then by default indeed the `_KEY` suffix is appended.